server/app/services/applicant/ReadOnlyApplicantProgramService.java

```diff
@@ -92,6 +101,9 @@ public interface ReadOnlyApplicantProgramService {
    */
   Optional<Block> getFirstIncompleteBlockExcludingStatic();
 
+  /** Returns summary data for each question in this application. */
```

Comment:
*suggestion*: This is only summary data for the hidden blocks, right, not for "each question"? This comment should specify that it only returns summary data for the hidden blocks.